> AssertionError: File contains trailing whitespace: /home/travis/build/sympy/sympy/sympy/polys/multivariate_resultants.py, line 139.

I have configured my editor (emacs) to show trailing whitespace in red. These errors can also be checked locally by `bin/test quality` if the editor does not show them.

--- a/sympy/polys/multivariate_resultants.py
+++ b/sympy/polys/multivariate_resultants.py
@@ -136,7 +136,7 @@ class DixonResultant():
                         useinstead="get_max_degrees",
                         issue=17763,
                         deprecated_since_version="1.5").warn()
-                        
+
         list_of_products = [self.variables[i] ** self.max_degrees[i]
                             for i in range(self.n)]
         product = prod(list_of_products)

--- a/sympy/polys/multivariate_resultants.py
+++ b/sympy/polys/multivariate_resultants.py
@@ -92,7 +92,7 @@ class DixonResultant():
     def max_degrees(self):
         SymPyDeprecationWarning(feature="max_degrees",
                         issue=17763,
-                        deprecated_since_version="1.5")
+                        deprecated_since_version="1.5").warn()
         return self._max_degrees
 
     def get_dixon_polynomial(self):

--- a/sympy/polys/multivariate_resultants.py
+++ b/sympy/polys/multivariate_resultants.py
@@ -137,7 +137,7 @@ class DixonResultant():
                         issue=17763,
                         deprecated_since_version="1.5").warn()
 
-        list_of_products = [self.variables[i] ** self.max_degrees[i]
+        list_of_products = [self.variables[i] ** self._max_degrees[i]
                             for i in range(self.n)]
         product = prod(list_of_products)
         product = Poly(product).monoms()

